src/test/java/org/prebid/server/cache/CacheServiceTest.java

```diff
@@ -351,7 +352,10 @@ public void cacheBidsOpenrtbShouldNeverCallCacheServiceIfNoBidsPassed() {
     public void cacheBidsOpenrtbShouldPerformHttpRequestWithExpectedTimeout() {
         // when
         cacheService.cacheBidsOpenrtb(singletonList(givenBidOpenrtb(identity())), singletonList(givenImp(identity())),
-                CacheContext.builder().shouldCacheBids(true).build(), account, timeout);
+                CacheContext.builder()
+                        .shouldCacheBids(true)
```

Comment:
If you are using '/n' for first parameter, keep this style for the next. 

**FIX for others**